.github/workflows/pipeline.yaml

```
@@ -0,0 +1,53 @@
+          ${{ runner.os }}-yarn-
+    - name: Cache node_modules
+      id: cache-node-modules
+      uses: actions/cache@v2
+      with:
+        path: node_modules
+        key: ${{ runner.os }}-${{ matrix.node-version }}-nodemodules-${{ hashFiles('**/yarn.lock') }}
+        restore-keys: |
+          ${{ runner.os }}-${{ matrix.node-version }}-nodemodules-
+    - name: Installing dependencies
+      run: yarn install --frozen-lockfile --prefer-offline
+      if: |
```

actionlint (1.7.4)

property "cache-yarn-cache" is not defined in object type {cache-node-modules: {conclusion: string; outcome: string; outputs: {string => string}}; yarn-cache: {conclusion: string; outcome: string; outputs: {string => string}}; yarn-cache-dir-path: {conclusion: string; outcome: string; outputs: {string => string}}}

(expression)